src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -763,27 +819,26 @@ void Transformations::Lpt(const std::vector<ov::element::Type>& defaultPrecision
     } else {
         input0LowPrecisionList = {ov::element::u8};
     }
+    std::vector<ov::element::Type> lowPrecisionList = {ov::element::u8, ov::element::i8};
 
     auto supportedPrecisions = std::vector<PrecisionsRestriction>({
         PrecisionsRestriction::create<ov::opset1::Convolution>({
             {{0}, input0LowPrecisionList},
             {{1}, {ov::element::i8}},
         }),
         PrecisionsRestriction::create<ov::opset1::ConvolutionBackpropData>(
-            {{{0}, {ov::element::u8, ov::element::i8}}, {{1}, {ov::element::i8}}}),
-        PrecisionsRestriction::create<ov::opset1::GroupConvolution>([input0LowPrecisionList](
+            {{{0}, lowPrecisionList}, {{1}, {ov::element::i8}}}),
+        PrecisionsRestriction::create<ov::opset1::GroupConvolution>([lowPrecisionList, input0LowPrecisionList](
                                                                         const std::shared_ptr<ov::Node>& node) {
             const auto& input_partial_shape = node->get_input_partial_shape(0);
             const auto& rank = input_partial_shape.rank();
             if (rank.is_static() && (rank.get_length() == 5)) {
-                return PrecisionsRestriction::PrecisionsByPorts{{{0}, {ov::element::u8, ov::element::i8}},
-                                                                {{1}, {ov::element::i8}}};
+                return PrecisionsRestriction::PrecisionsByPorts{{{0}, lowPrecisionList}, {{1}, {ov::element::i8}}};
             }
 
             return PrecisionsRestriction::PrecisionsByPorts{{{0}, input0LowPrecisionList}, {{1}, {ov::element::i8}}};
         }),
-        PrecisionsRestriction::create<ov::opset1::MatMul>(
-            {{{0}, {ov::element::u8, ov::element::i8}}, {{1}, {ov::element::i8}}}),
+        PrecisionsRestriction::create<ov::opset1::MatMul>({{{0}, lowPrecisionList}, {{1}, {ov::element::i8}}}),
```

Comment:
Lets avoid unnecessery changes in non-arm part

Reply:
Reverted